src/components/StoryList.component.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from 'react'
+import { Button, FormControl } from 'react-bootstrap'
+import Sidebar from 'react-sidebar'
+import './../styles/storylist.css'
+
+class StoryList extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      addStorySelected: false,
+      storyName: ''
+    }
+
+    this.addStoryClicked = this.addStoryClicked.bind(this)
+    this.addStoryExit = this.addStoryExit.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+    this.submitStoryName = this.submitStoryName.bind(this)
+  }
+
+  addStoryClicked() {
+    this.setState({ addStorySelected: true })
+  }
+
+  addStoryExit() {
+    this.setState({ addStorySelected: false })
+  }
+
+  handleChange(e) {
+    this.setState({ storyName: e.target.value })
+  }
+
+  submitStoryName() {
+    this.setState({
+      addStorySelected: false,
+      storyName: ''
+    })
+    //make api call here maybe?
+  }
+
+  render() {
+    const stories = this.props.stories
+
+    const sidebarContent = (
+      <SidebarContent
+        {...this.props}
+        addStorySelected={this.state.addStorySelected}
+        storyName={this.state.storyName}
+        stories={stories}
+        addStoryClicked={this.addStoryClicked}
+        addStoryExit={this.addStoryExit}
+        handleChange={this.handleChange}
+        submitStoryName={this.submitStoryName}
+      />
+    )
+
+    const sidebarProps = {
```

Comment:
Since the return value of the `render` function is really small, as well as the props, it's fine to just send these props in as normal instead of separating them out into this object